docker/Dockerfile.test

```diff
@@ -1,42 +1,30 @@
-FROM datajoint/djbase:py3.9-debian-fcd8909
+FROM datajoint/djbase:py3.9-debian-8eb1715
 
+# ARG GITHUB_USERNAME=datajoint # tried moving to ENV
 USER anaconda:anaconda
 
 COPY ./workflow-deeplabcut/docker/apt_requirements.txt /tmp/
 RUN /entrypoint.sh echo "Installed dependencies."
 
 WORKDIR /main/workflow-deeplabcut
 
-# Option 1 - Install DataJoint's remote fork of the workflow and elements
-# RUN git clone https://github.com/datajoint/workflow-deeplabcut.git /main/
-
-# Option 2 - Install user's remote fork of element and workflow
-#            or an unreleased version of the element
-# RUN pip install git+https://github.com/<user>element-lab.git
-# RUN pip install git+https://github.com/<user>/element-animal.git
-# RUN pip install git+https://github.com/<user>/element-session.git
-# RUN pip install git+https://github.com/<user>/element-deeplabcut.git
-# RUN git clone https://github.com/<user>/workflow-deeplabcut.git /main/workflow-deeplabcut
-
-# Option 3 - Install user's local fork of element and workflow
-RUN mkdir /main/element-lab \
-          /main/element-animal \
-          /main/element-session \
-          /main/element-deeplabcut \
-          /main/workflow-deeplabcut
-
-COPY --chown=anaconda:anaconda ./element-lab /main/element-lab
-COPY --chown=anaconda:anaconda ./element-animal /main/element-animal
-COPY --chown=anaconda:anaconda ./element-session /main/element-session
-COPY --chown=anaconda:anaconda ./element-deeplabcut /main/element-deeplabcut
-COPY --chown=anaconda:anaconda ./workflow-deeplabcut /main/workflow-deeplabcut
-
-RUN pip install -e /main/element-lab
-RUN pip install -e /main/element-animal
-RUN pip install -e /main/element-session
-RUN pip install -e /main/element-deeplabcut
-RUN rm -f /main/workflow-deeplabcut/dj_local_conf.json
-
-# Install the workflow
-RUN pip install /main/workflow-deeplabcut
-RUN pip install -r /main/workflow-deeplabcut/requirements_test.txt
+# Always get interface/djarchive
+RUN pip install --no-deps "element-interface@git+https://github.com/datajoint/element-interface"
+RUN pip install --no-deps "djarchive-client@git+https://github.com/datajoint/djarchive-client"
```

Comment:
I would suggest these guys to be installed from the `requirements_test.txt`.

Reply:
But I see that that file is removed. Never mind then.

Reply:
I think there is a general inconsistency across the workflows. These pytests only work in the docker environment but not in the pip install conditions. This is something to be fixed latert

Reply:
The pytests work for me in my host machine if I configure `TEST_DATA_DIR` in `docker/.env`. My goal with this docker/pytest revamp was to make them more configurable on launch. Rather than going into `tests/__init__.py` to toggle teardown or going into `Dockerfile.test` to set install source, they can be done from the pytest command or `env` files.